src/modules/history/components/HistoryOptionsList.tsx

```diff
@@ -53,6 +55,16 @@ export const HistoryOptionsList: React.FC = () => {
 	return (
 		<Drawer classes={{ paper: 'history-options-sidebar' }} anchor="left" variant="permanent">
 			<Toolbar />
+			<Tooltip title={I18N.translate('profileDescription')}>
+				<Box textAlign="center">
+					<Typography variant="overline">
+						<Box display="inline" fontWeight="fontWeightBold">
+							{I18N.translate('profile')}:
+						</Box>{' '}
+						{api?.session?.profileName || I18N.translate('unknown')}
```

Comment:
I think this should be highlighted a lot! Perhaps even moved to the main window.

Reply:
Any suggestions? Moved to above the items?

Reply:
Yeah, above the items. Preferable in the blue header bar. Centered: So it is always on top. The text could be something like: "History sync for {streamingService}, user: {profileName}

Reply:
Done. I used "profile" instead of "user" because it's already translated.

![](https://user-images.githubusercontent.com/25509361/129397254-2fdcc80e-889c-4a9a-aa49-a7146d255b60.png)